holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,128 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from gettext import gettext as tr
+
+from holidays import FEB
+from holidays.groups import (
+    ChristianHolidays,
+    IslamicHolidays,
+    InternationalHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class IvoryCoast(
+    ObservedHolidayBase, ChristianHolidays, IslamicHolidays, InternationalHolidays, StaticHolidays
+):
+    """Ivory Coast holidays.
+
+    References:
+        * <https://natlex.ilo.org/dyn/natlex2/r/natlex/fe/details?p3_isn=44374>
+        * <https://www.droit-afrique.com/uploads/RCI-Decret-1996-205-jours-feries.pdf>
+        * <https://fr.wikipedia.org/wiki/F%C3%AAtes_et_jours_f%C3%A9ri%C3%A9s_en_C%C3%B4te_d%27Ivoire>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Ivory_Coast>
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observé, estimé)")
```

Comment:
```suggestion
    # Day after the %s.
    observed_label = tr("Lendemain de la %s")
    # Day after the %s (estimated).
    observed_estimated_label = tr("Lendemain de la %s (estimé)")
```
as it's more consistent with the official text (with the corresponding translations in .po files).

Reply:
I guess this works as well, better than outright setting islamic observance label to False by default 👍